cranelift/filetests/filetests/runtests/simd-arithmetic-nondeterministic2.clif

```diff
@@ -0,0 +1,28 @@
+; Test the non-deterministic aspects of the SIMD arithmetic operations.
+; If you change this file, you should most likely update
+; simd-arithmetic-nondeterministic.clif as well.
+test run
+set enable_simd
+target x86_64 machinst skylake
```

Comment:
Is this the minimum necessary or would `nehalem` work too? Last time I checked the macOS builders on GHA were still at `nehalem`.

Reply:
Honestly, I don't know; I just copied the value from `simd-arithmetic.clif`.

Reply:
It's probably not too important, as tests such as this are OS-independent and will get coverage on the (more up-to-date-CPU) Linux runners.